cmd/soci/commands/ztoc/info.go

```diff
@@ -30,15 +32,23 @@ import (
 )
 
 type Info struct {
-	Version   string     `json:"version"`
-	BuildTool string     `json:"build_tool"`
-	Files     []FileInfo `json:"files"`
+	Version           string             `json:"version"`
+	BuildTool         string             `json:"build_tool"`
+	Size              int64              `json:"size"`
```

Comment:
The `Size` is the artifact size right (ztoc blob size in db)?

Reply:
Yes.

Reply:
Yeah I don't believe this change will break my integ test (unmarshal should just skip missing fields). I would have to update the current verification func which compares the fields values with the original ztoc.